about.html

```diff
@@ -0,0 +1,161 @@
+            </defs>
+        </svg>
+    </section>
+    </div>
+
+    <div class="past-cover">
+    <section class="past">
+        <h2>See the Past Global Soccer Summits</h2>
+        <div class="underline"></div>
+        <div class="past-cover">
+        <div class="hero background-overlay">
+            <h3>2019</h3>
+            <p>Global soccer summit 2019, Moscow</p>
+        </div>
+        <div class="sep"></div>
+        <div class="hero conf background-overlay">
+            <h3>2021</h3>
+            <p>Global soccer summit 2019, Windhoek</p>
+        </div>
+        </div>
+    </section>
+    </div>
+
+    <section class="mobile-partners">
+        <hr>
+        <h2>Partners</h2>
+        <div class="underline"></div>
+        <div class="partners-wrap">
+           <img src="images/mozilla-logo.PNG" alt="Google logo">
+           <img src="images/google-logo.PNG" alt="Kakao logo">
+           <img src="images/daum-logo.PNG" alt="lumen logo">
+        </div>
+        <div class="partners-sub">
+           <img src="images/naver-logo.PNG" alt="reho logo">
+           <img src="images/air-logo.PNG" alt="mozilla logo">
+        </div>
+    </section>
+
+    <section class="partners">
+        <h2>Partners</h2>
+        <div class="underline"></div>
+        <div class="part-cont">
+            <img src="images/mozilla-logo.PNG" alt="Google logo">
+            <img src="images/google-logo.PNG" alt="Kakao logo">
+            <img src="images/daum-logo.PNG" alt="lumen logo">
+            <img src="images/naver-logo.PNG" alt="reho logo">
+            <img src="images/air-logo.PNG" alt="mozilla logo">
+        </div>
+    </section>
+
+    <section class="footer" id="about-footer">
+        <div class="container">
+            <img src="images/cckfoot.PNG" alt="soccer summit logo">
+            <p>2023 Global soccer summit<br> Kisumu International
+            sports zone Arena. </p>
+        </div>
+    </section>
```

Comment:
Great design @marvin-nyalik 🤝

- [ ] Kindly ensure your design replicates the design given as much as possible.
- Your design
![Your-about-footer](https://github.com/marvin-nyalik/SoccerFestival/assets/63917725/c2dc7df8-a867-41a8-b2d3-ec4a4a1ae45c)
- Given design
![Given-about-footer](https://github.com/marvin-nyalik/SoccerFestival/assets/63917725/5eff63d3-c0ee-4460-b8cb-e4b3c3e316b2)